backend/apid/middlewares/allowlist_test.go

```diff
@@ -57,7 +59,7 @@ func TestMissingTokenFromAllowList(t *testing.T) {
 	// Perform the request with the middleware
 	res, err := http.DefaultClient.Do(req)
 	assert.NoError(t, err)
-	assert.Equal(t, http.StatusUnauthorized, res.StatusCode)
+	assert.Equal(t, res.StatusCode, http.StatusUnauthorized)
```

Comment:
The documentation appears that the order is expected, then actual: https://godoc.org/github.com/stretchr/testify/assert#Equal
```suggestion
	assert.Equal(t, http.StatusUnauthorized, res.StatusCode)
```

Reply:
🤦‍♂️